skorch/tests/test_regressor.py

```diff
@@ -90,4 +107,8 @@ def test_score(self, net, data):
         X, y = data
         r2_score = net.score(X, y)
         assert r2_score <= 1.
-        return r2_score
+
+    def multi_target_test_score(self, multi_target_net, multi_target_data):
+        X, y = multi_target_data
+        r2_score = multi_target_net.score(X, y)
```

Comment:
Before scoring, you still need to call `fit`.

Reply:
Ok but I did not do it for the single output regression and for the classification. Should I do it as well?

Reply:
No need, the other nets are already fitted at this point, only this one isn't.